wgengine/netstack/netstack.go

```diff
@@ -350,7 +350,8 @@ func (ns *Impl) forwardTCP(client *gonet.TCPConn, wq *waiter.Queue, port uint16)
 		}
 		cancel()
 	}()
-	server, err := ns.DialContextTCP(ctx, net.JoinHostPort("localhost", strconv.Itoa(int(port))))
+	dialer := &net.Dialer{}
```

Comment:
Let's name it very explicitly so we don't confuse ourselves/otherwise in the future:

```go
var stdDialer net.Dialer
stdDialer.DialContext(...)
```